packages/kbn-test/src/kbn_client/kbn_client_import_export.ts

```diff
@@ -56,6 +56,31 @@ export class KbnClientImportExport {
     return this.dir ? Path.resolve(this.dir, path) : path;
   }
 
+  async bulkCreate(name: string, options?: { space?: string }) {
```

Comment:
I extended the `KbnClient` to provide a `bulkCreate` method. It behaves the same as `import`, but it uses `bulkCreate` under the hood :smile: